release-drafter.yml

```diff
@@ -0,0 +1,19 @@
+name: Release-drafter
+
+on:
+#   push:
+#     branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Should this be push on master branch which would run after a pull request is squashed and merged?

Reply:
Resolved.

Reply:
Are both the trigger needed ?

Reply:
I believe so. When a PR gets checked in on master branch, the change should also be reflected in release notes.

Reply:
Can you validate this behavior? AFAIK pull_request will trigger when pull request is created against the master branch. We want the release notes to be updated only after the pull request is merged to master (which results in a push on master branch). 

Check release drafter: https://github.com/release-drafter/release-drafter#usage